rowhammer_tester/scripts/rowhammer.py

```diff
@@ -32,6 +34,10 @@ def __init__(
             setattr(self, name, val)
         self.converter = DRAMAddressConverter.load()
         self._addresses_per_row = {}
+        self.bitflip_found = False
+        self.prompt_stop_test = True
+        self.do_error_summary = True
```

Comment:
This may become a `log_directory` path that is set to `None` by default. It can then be used to output the files in a chosen directory (that can be changed by an additional argument, e.g. `--log-dir`.

This will allow storing the log files, such as error reports, in a user-defined location, rather than the script execution location.